meson.build

```diff
@@ -176,6 +162,7 @@ src_files = files(
     'src/ALabel.cpp',
     'src/AIconLabel.cpp',
     'src/AAppIconLabel.cpp',
+    'src/modules/bluetooth.cpp',
```

Comment:
This wasn't guarded by any OS-specific conditions (`-Dlogind` doesn't count as one), but somehow I doubt bluez is available on various BSD systems.
Maybe `src/modules/bluetooth.cpp` should go under `is_linux` to avoid confusion. @jbeich, what do you think?

Reply:
should definitely be under is_linux